fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java

```diff
@@ -580,7 +580,6 @@ protected void unprotectedExecuteCancel(FailMsg failMsg, boolean abortTxn) {
             }
         }
         idToTasks.clear();
-        loadStatistic.clearAllLoads();
```

Comment:
is it OK to delete this?

Reply:
It OK, there is only some informations for showing.
And I just want to see how many rows it read after being cancelled.